flowmachine/tests/test_subscriber_handset_stats.py

```diff
@@ -0,0 +1,55 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# -*- coding: utf-8 -*-
+
+from flowmachine.features.subscriber.handset_stats import *
+
+import pytest
+
+from flowmachine.features import SubscriberHandsets
+
+# the test db contains no handset information so we can only test if the query
```

Comment:
Test data does include a teeny tiny fake tac db. Augment that so this can actually be tested?

Reply:
Fake data added.